similarity_api/src/settings.py

```diff
@@ -25,3 +28,7 @@ def get_postgres_credentials_from_vcap():
 SQLALCHEMY_DATABASE_URI = f"postgresql://{DB_USERNAME}:{DB_PASSWORD}@{DB_HOST}:{PORT}/{DB_NAME}"
 
 API_KEY = os.environ.get("SIMILARITY_API_KEY")
+
+SCHEDULER_JOBSTORES = {
+  'default': SQLAlchemyJobStore(url='sqlite:///jobs.sqlite')
```

Comment:
Should we add this file to .gitignore?

Reply:
what is the path to the file? does it need to live across executions? could it be placed in a location that is always destroyed when the container is relaunched

Reply:
`similarity_api/src/jobs.sqlite`. It does not need to live across deploys or redeploys. It is created if it doesn't exist, and when the app starts for the first time, the job is deleted from the DB if it exists and is recreated. In other words, we can safely not deploy this file. I added it to gitignore here: c26190b06c09b5594d1596abe9ec0ccf1e133c04